settings.py

```diff
@@ -70,7 +70,7 @@
 ES_DEFAULT_NUM_REPLICAS = 0
 
 SITE_URL = os.environ.get('OLYMPIA_SITE_URL') or 'http://localhost:8000'
-SERVICES_DOMAIN = 'localhost:8000'
+SERVICES_DOMAIN = SITE_URL.replace('http://', '').replace('https://', '')
 SERVICES_URL = 'http://%s' % SERVICES_DOMAIN
```

Comment:
Oh, I guess it's for local dev and is replaced elsewhere, in which case makes sense.